docs/src/templates/Overview.tsx

```diff
@@ -30,7 +30,8 @@ const Overview = ({ location, pageContext }: Props) => {
         columns="1fr"
         gap="2rem"
         largeMobile={{ columns: "repeat(2, 1fr)" }}
-        tablet={{ columns: "repeat(3, 1fr)" }}
+        desktop={{ columns: "repeat(3, 1fr)" }}
+        largeDesktop={{ columns: "repeat(2, 1fr)" }}
```

Comment:
I forgot about this. Merge master in `master` to get latest table of contents changes, then use the `useTableOfContents` hook to check whether there are any items (no items = no table of contents), then use three columns if there aren't any, otherwise two columns:

```tsx
const [tableOfContents] = useTableOfContents()

// inside Grid props
{...(tableOfContents.length > 0 ? {
  desktop: { columns: "repeat(3, 1fr)" },
  largeDesktop: { columns: "repeat(2, 1fr)" },
} : {
  tablet: { columns: "repeat(3, 1fr)" },
})}
```